examples/process-application/src/main/kotlin/io/holunda/camunda/taskpool/example/process/ExampleProcessApplication.kt

```diff
@@ -89,4 +89,35 @@ class ExampleProcessApplication {
     })
   */
 
+
+  @Bean
+  @Primary
+  fun myTaskCommandErrorHandler(): TaskCommandErrorHandler = object : LoggingTaskCommandErrorHandler(logger) {
+    override fun apply(commandMessage: Any, commandResultMessage: CommandResultMessage<out Any?>) {
+      logger.error { "<--------- CUSTOM ERROR HANDLER REPORT --------->" }
+      super.apply(commandMessage, commandResultMessage)
+      logger.error { "<------------------- END ----------------------->" }
+    }
+  }
+
+  /*
```

Comment:
Did you leave this commented code here on purpose? Maybe add a comment like _Uncomment this to..._